identity.html

```diff
@@ -902,7 +931,317 @@ <h2><dfn>RTCIdentityAssertion</dfn> Attributes</h2>
             </dd>
           </dl>
         </section>
+       </div>
+      </section>
+      <section>
+        <h4>Setting the RTCPeerConnection configuration</h4>
```

Comment:
Not sure about use of Title Case or Sentence case for headings:
```suggestion
        <h4>Setting the RTCPeerConnection Configuration</h4>
```